content/md/imports-exports/tailored-import.md

```diff
@@ -142,17 +142,23 @@ Depending on the targets that you define, here are the different types of operat
 
 ## Yes/No replacement
 
-Replace the default values with your own value (only available for the "Yes/No" attribute or the "Enabled" system property).
+This operation is available for `Yes/No` and `Enabled` attributes used as a target.
+It will help you to replace the default values with your own value.
 ![Yes/No replacement operation](../img/Tailored-Import_Yes-No_Replacement.png)
 
 ## Replacement
 
-Replace the values by your own (only available for simple select, multi select<!--, reference entities single link or multiple link--> attributes).
+Replace the values by your own. Only available for simple select, multi select, reference entities single link<!--or multiple link--> attributes.
 ![Replacement operation](../img/Tailored-Import_Replacement.png)
 
+::: warning
+Replacement values are case-sensitive.
+:::
+
 ## Split
 
-If you want to import different values that are located into the same cell of your spreadsheet, you can use this operation to split the values thanks to a separator: comma (,), semicolon (; ), pipe (|).
+This operation is available for collection attributes used as a target.
+It will help you to import different values that are located into the same cell of your spreadsheet by splitting these values thanks to a separator: comma (,), semicolon (; ) or pipe (|).
```

Comment:
```suggestion
It will help you to import different values that are located into the same cell of your spreadsheet by splitting these values thanks to a separator: comma (,), semicolon (;) or pipe (|).
```

Reply:
On purpose, as in front it generates an emoji instead...